src/Core/Usecase/DataProvider/UpdateDataProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Ushahidi Platform Utilize Data Provider
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2015 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Usecase\DataProvider;
+
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\Usecase\UpdateUsecase;
+
+class UpdateDataProvider extends UpdateUsecase
```

Comment:
We don't actually ever update data providers. We enable them in dataprovider config..